### DISCUSSION TITLE
Newbie trying to learn, is this safe?

### DISCUSSION THREAD

**Discussion by GonzaPicks** [community/community#36718] [Other Feature Feedback, Questions, & Ideas]
hey, I was taking the cs50 edu course and to submit my problems it directed me here. I had to follow this guide https://cs50.readthedocs.io/github/#ssh but to be honest I have no idea what I'm doing. Is this safe or I'm actually granting some access I shouldn't grant?. Just want to know if I should be worried. I actually followed the steps and after typing yes on the terminal, it showed loads of "y" and couldn't do nothing else on the terminal. Thanks!

**Comment by ossanna16** [comment]
Hi there @GonzaPicks and welcome to our community! This is definitely a safe place for you to ask question. It would help if you included more details for your question such as screenshots, etc.
